### ISSUE TITLE
Spanish translation is in need of thorough revision 

### ISSUE THREAD

**Issue by NikoVP** [diasurgical/devilutionX#1872]
The Spanish translation is in dire need of revision. It possibly has been generated with machine translation, littered with literal translation, spelling mistakes, misstranslated terms and frases that don't make sense . Needs to be revised from a native speaker. Also probably should be split for European Spanish and Latin American Spanish. 

**To Reproduce**
1. Go to translation folder in repository 
2. Click on es.po
3. Become sad

**Expected behavior**
Should be well written, condensed (since Spanish is a wordy language) and easy to read.

**Comment by Cowcat5150** [comment]
Example: "journal" is translated as "revista". In "modern European Spanish" it means "magazine". The correct should be "diario" as diary in plain English. Don't know if in L.A. Spanish they keep the same way.

I probably can look at that but the issue is that the game is located in medieval times, so translation should be tailored to that or not? I'm old but not that far.

**Comment by NikoVP** [comment]
> Example: "journal" is translated as "revista". In "modern European Spanish" it means "magazine". The correct should be "diario" as diary in plain English. Don't know if in L.A. Spanish they keep the same way.
> 
> I probably can look at that but the issue is that the game is located in medieval times, so translation should be tailored to that or not? I'm old but not that far.
Absolutely it should be tailored and not copy-pasted from google translate. One example from the said file
```
msgid "Quarter Staff"
msgstr "Personal del barrio"
```

**Comment by hiperiondev** [comment]
Yes, a lot of translation is made with automatic tools and must be revised. Is a lot of text. Somebody can help?

**Comment by hiperiondev** [comment]
When I started collaborating with the translation, there was absolutely nothing. One part was made by hand and another part automatically. Still, with the free time I have, it took me almost a month to complete it. My intention is to have something fairly complete that is easier to correct in the shortest possible time. Also, in my experience, more people are more likely to join a project already started than to expect them to do so if there is nothing yet.
I hope this is the start of more people helping with the translation. It would also be good to open a separate discussion in Spanish to be able to freely discuss in our native language.
The issue of the Saxon Genitive also remains to be resolved.

**Comment by hiperiondev** [comment]
Now I will start to correct all the text by hand, which will take a long time.

**Comment by AJenbo** [comment]
>  The issue of the Saxon Genitive also remains to be resolved.

You never replied with a description of what the problem is

**Comment by NikoVP** [comment]
> > The issue of the Saxon Genitive also remains to be resolved.
> 
> You never replied with a description of what the problem is

I think he is referring to how item prefixes should be handled in Spanish since thr language does not accept them being infront of the item. Hopefully a native speaker should offer a suitable solution.

**Comment by AJenbo** [comment]
We will soon allow handeling of that via "{prefix} {itemName}" being translated to "{prefix} {itemName}", for more advanced rules we plan for allowing translators to make a short LUA script for handling generating item names in a way that comports with there language rules.

**Comment by hiperiondev** [comment]
> I think he is referring to how item prefixes should be handled in Spanish since the language does not accept them being infront of the item. 

Exactly. I think I remember that this problem was already discussed in the last modification of the Italian translation

**Comment by julealgon** [comment]
> When I started collaborating with the translation, there was absolutely nothing. One part was made by hand **and another part automatically.**

Did you make this point clear at the time of the PR @hiperiondev ? If I saw machine translated text and was aware of it, I'd never have approved a PR with that.

From discussions in discord, it seemed like many people were not aware there was machine translated text without any double checking in the Spanish translation.

**Comment by hiperiondev** [comment]
> > When I started collaborating with the translation, there was absolutely nothing. One part was made by hand **and another part automatically.**
> 
> Did you make this point clear at the time of the PR @hiperiondev ? If I saw machine translated text and was aware of it, I'd never have approved a PR with that.
> 
> From discussions in discord, it seemed like many people were not aware there was machine translated text without any double checking in the Spanish translation.

No, that was my mistake, I really apologize. If you want to delete it and wait for it to be fully corrected, I have no problems.
Right now I am correcting everything by hand but it is a lot of text, it will take a while.
If someone else wants to collaborate it would be much faster but, so far, I am doing it alone

**Comment by AJenbo** [comment]
If they where machine translations with little to no validation then they should have been marked as "needs work"/"fuzzy" in the translation file.

**Comment by hiperiondev** [comment]
I did not know, it is my first translation. I'll start marking it that way

**Comment by hiperiondev** [comment]
[es.zip](https://github.com/diasurgical/devilutionX/files/7026003/es.zip)

56% corrected.
All non corrected marked for revision
A lot of comments about translation (missing contexts or unknown correct translations)

Still working... slowly but surely

**Comment by AJenbo** [comment]
Nice :)

If you can submit it as a pull request that would be ideal so that you get the credit, and reviews hand be handled better.

**Comment by hiperiondev** [comment]
Done! 

https://github.com/diasurgical/devilutionX/pull/2665

**Comment by hiperiondev** [comment]
All shot texts corrected: https://github.com/diasurgical/devilutionX/pull/2675

**Comment by hiperiondev** [comment]
There is still the problem of the Saxon genitive. All texts in Latin languages will look bad.

Another problem is that of gender. For example "Welcome to the" in Spanish will depend on the gender of the word that follows ("the" can be "el" or "la")

**Comment by AJenbo** [comment]
> There is still the problem of the Saxon genitive. All texts in Latin languages will look bad.

Yeah, as mentioned this won't be possible to address before we have LUA intergration.

> Another problem is that of gender. For example "Welcome to the" in Spanish will depend on the gender of the word that follows ("the" can be "el" or "la")

This is a bit tricky to change, but we are looking in to it.

**Comment by AJenbo** [comment]
> Done!
> 
> #2665

There is still some work to be done before the Spanish translation is considered complete, but I think this took care of the issue defined here.
